keras/src/backend/openvino/excluded_concrete_tests.txt

```diff
@@ -81,7 +81,6 @@ NumpyOneInputOpsCorrectnessTest::test_conj
 NumpyOneInputOpsCorrectnessTest::test_correlate
```

Comment:
please also remove `NumpyDtypeTest::test_diag` line

Reply:
you can see a line ` NumpyDtypeTest::test_diag` above in this file. Please remove it as well to switch on tests